deploy/mod/deploy.py

```diff
@@ -669,6 +685,7 @@ def make_osd_fs(self, osd, osd_num, osd_device, journal_device, mount_list):
         osd_filedir = osd_filename.replace("$id", str(osd_num))
         common.pdsh( user, [osd], 'mkdir -p %s/%s' % (osd_basedir, osd_filedir))
         common.pdsh( user, [osd], 'mount %s -t xfs %s %s/%s' % (mount_opts, osd_device, osd_basedir, osd_filedir))
+        self.make_osd(osd, osd_num, osd_device, journal_device)
```

Comment:
Oh, I see, I was misleading by the github page, thought you put make_osd into "make_osd_ceph_disk_prepare" function.
Anyhow, I am still concerned about if you simply uses multi thread to do the make_osd work, it may bring some un desired osd order in osd_map. Did you check osd_tree after the deployment? 

Is it possible, you move make_osd out of make_osd_fs function, and after all make_osd_fs works done, then start do make_osd one by one? That makes more sense.